history/cultures/elf_group.txt

```diff
@@ -76,9 +79,13 @@
 	discover_innovation = innovation_development_04
 	#
 }
+#Using 603 for this tech
+603.1.1 = {
```

Comment:
The year is wrong. We have the Dates linked on our documentation.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Coder-Guidelines#documentations